base/BUILD.gn

```diff
@@ -3322,7 +3322,7 @@ test("base_unittests") {
     # "profiler/stack_sampling_profiler_unittest.cc",
     "rand_util_unittest.cc",
 
-    # "ranges/algorithm_unittest.cc",
+    "ranges/algorithm_unittest.cc",
```

Comment:
It looks like at least this doesn't build, see https://github.com/youtube/cobalt/actions/runs/7217173309/job/19664596090?pr=2092#step:6:1669